atm.js

```diff
@@ -1,21 +1,77 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//Global Vars
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var currentCheckBalance = 0;
+var currentSaveBalance = 0;
 
-}
+// event listeners
 
-function checkingWithdrawal(amount){
+// checking deposit button
 
-}
+$('#checkDepositButton').click('click', function(){
+    var deposit = $('#checkingInput').val();
+    currentCheckBalance = currentCheckBalance + parseInt(deposit);
+    $('#checkBalanceWindow').text('$' + currentCheckBalance);
 
-function savingsDeposit(amount){
+    // zero balance color change
+    if (currentCheckBalance == 0){
+        $('#checkBalanceWindow').css('background-color', 'red');
+    }
+    else if (currentCheckBalance > 0){
+        $('#checkBalanceWindow').css('background-color', '');
+    }
+});
 
-}
+// withdraw deposit button
+
+$('#checkWithdrawButton').click('click', function(){
+    var withdraw = $('#checkingInput').val();
+    if (currentCheckBalance >= withdraw){
+        currentCheckBalance = currentCheckBalance - parseInt(withdraw);
+        $('#checkBalanceWindow').text('$' + currentCheckBalance);
+    }
```

Comment:
You mentioned having trouble with the overdraft protection - but I don't see any code here for it. Let me know if you'd like to go over the overdraft solution in a one on one!